packages/react-components/react-combobox/stories/Combobox/ComboboxVirtualizer.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Combobox, Option, makeStyles, useId } from '@fluentui/react-components';
+import type { ComboboxProps } from '@fluentui/react-components';
+
+import { Virtualizer, useStaticVirtualizerMeasure } from '@fluentui/react-components/unstable';
+
+const useStyles = makeStyles({
+  listbox: {
+    maxHeight: '250px',
+  },
+  option: {
+    height: '32px',
+  },
+});
+
+export const ComboboxVirtualizer = (props: Partial<ComboboxProps>) => {
+  const comboId = useId('combobox');
+
+  const itemHeight = 32; //This should match the height of each item in the listbox
+  const numberOfItems = 10000;
+
+  const { virtualizerLength, bufferItems, bufferSize, scrollRef } = useStaticVirtualizerMeasure({
+    defaultItemSize: itemHeight,
+    direction: 'vertical',
+  });
+
+  const styles = useStyles();
+
+  return (
+    <div>
+      <div>
+        <label htmlFor={`${comboId}`}>Medium</label>
+        <Combobox
+          aria-labelledby={`${comboId}`}
```

Comment:
small nit, this should actually be `id` (or alternatively, keep the `aria-labelledby` here and put `id` on the label, but I'd prefer the native htmlFor/id pair)

```suggestion
          id={`${comboId}`}
```